### ISSUE TITLE
[Bug] blazorwasm - hosted b2c authority needs to include the policy

### ISSUE THREAD

**Issue by jennyf19** [AzureAD/microsoft-identity-web#485]
The client in the blazorwasm-hosted is using MSAL JS, and it needs to have the policy appended to the end in order for the OIDC metadata endpoint to be discovered.

```csharp
 "AzureAdB2C": {
        "Authority": "https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/b2c_1_susi/",
        "ClientId": "25f7fc07-bb5b-49ad-8179-6d486d094ca6",
        "ValidateAuthority": false
    }
```

This `"https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/` does not work and results in an OIDC metadata discovery error.

**Comment by jmprieur** [comment]
This is actually a tool problem (needs to be fixed here):
https://github.com/AzureAD/microsoft-identity-web/blob/1bb75bd2a0ec532ba35f07360b53ed294e915cea/ProjectTemplates/configuration.json#L5

The template is right:
https://github.com/AzureAD/microsoft-identity-web/blob/master/ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Client/wwwroot/appsettings.json#L10

**Comment by jennyf19** [comment]
Included in [0.3.0-preview release](https://github.com/AzureAD/microsoft-identity-web/releases/tag/0.3.0-preview).
